docs/src/pages/components/data-grid/sorting/sorting.md

```diff
@@ -4,34 +4,91 @@ title: Data Grid - Sorting
 
 # Data Grid - Sorting
 
-<p class="description">Sorting allows ordering records in the data grid.</p>
+<p class="description">Easily sort your rows based on one or several criteria.</p>
 
-## Basic sorting
+The sorting rules can be modified through the grid interface by clicking on a column header. Repeat this action to pass to the next sorting direction of the column.
 
-Single column sorting can be triggered with by clicking a column header.
-Repeat this action to change the sorting direction.
+{{"demo": "pages/components/data-grid/sorting/BasicExampleDataGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
-A sorted column can be can pre-configured using the `sortModel` prop:
+## Single and multi-sorting
 
-{{"demo": "pages/components/data-grid/sorting/BasicSortingGrid.js", "bg": "inline"}}
+> ⚠️ The `DataGrid` can only sort the rows according to one criterion at the time.
+>
+> To use multi-sorting, you need to upgrade to the [Pro plan](https://mui.com/store/items/material-ui-pro/)
+
+The demo below let you sort the rows according to several criteria at the same time.
+
+Hold down the <kbd class="key">CTRL</kbd> or <kbd class="key">Shift</kbd> (use <kbd class="key">⌘ Command</kbd> on macOS) key while clicking the column header.
+
+{{"demo": "pages/components/data-grid/sorting/BasicExampleDataGridPro.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Pass sorting rules to the grid
+
+### Structure of the model
+
+The sort model is a list of sorting item.
+Each item represent a sorting rule and is composed of several elements:
+
+- `sortingItem.field`: the field on which we want to apply the rule
+- `sortingItem.sort`: the direction of the sorting (`'asc'`, `'desc'`, `null` or `undefined`, if `null` or `undefined`, the rule will not be applied)
+
+### Initialize the sort model
+
+To initialize the sort model without controlling it, provide the model to the `initialState` prop.
+
+```jsx
+<DataGrid
+  initialState={{
+    sorting: {
+      sortModel: [{ field: 'rating', sort: 'desc' }],
+    },
+  }}
+/>
+```
+
+{{"demo": "pages/components/data-grid/sorting/InitialSort.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Controlled sort model
+
+Use the `sortModel` prop to control the sorting rules applied on the rows.
+
+You can use the `onSortModelChange` prop to listen to changes to the sorting rules and update the prop accordingly.
```

Comment:
```suggestion
You can use the `onSortModelChange` prop to listen to changes in the sorting rules and update the prop accordingly.
```
To avoid to to to